tests/infra_test.go

```diff
@@ -783,28 +784,39 @@ var _ = Describe("[Serial]Infrastructure", func() {
 			}
 		})
 
-		It("[test_id:4146]should include VMI phase metrics for few running VMs", func() {
-			// this tests requires at least two running VMis. To ensure this condition,
-			// the simplest way is just always run an additional VMI.
-			By("Creating another VirtualMachineInstance")
+		Context("Creating another VirtualMachineInstance", func() {
```

Comment:
I do not understand the context description. it is not suppose to tell me how it implements it, but what it represents.

Context text cannot replace By text.